### ISSUE TITLE
ClaferMooVisualizer does not work with clafer 0.3.9

### ISSUE THREAD

**Issue by toeklk** [gsdlab/ClaferMooVisualizer#17]
Hi,

I was trying to get the ClaferMooVisualizer from the develop branch working with the claferChocoIG code from the development branch.  At first sight, this seems to work fine, as one can see in the output window (example: mobile phone example)
```
ClaferMooVisualizer> clafer --ss=none file.cfr {mode args}
Compiler> Working
All clafers: 23 | Abstract: 2 | Concrete: 21 | Reference: 2
Constraints: 27
Goals: 1
Global scope: 1..*
Can skip name resolver: no


Compiler> Model's XML got successfully
ClaferMooVisualizer> java -jar /Backends/../../../ClaferChocoIG/target/claferchocoig-0.3.9-jar-with-dependencies.jar --file=file.js --maxint=127 --scope=25
ClaferMooVisualizer> Backend output:
Clafer Choco Instance Generator and Multi-Objective Optimizer
Running Model...
Generating instances...
Generated 4 instance(s)
ClaferMooVisualizer> Backend output ends
ClaferMooVisualizer> Backend successfully executed
ClaferMooVisualizer> Updating the views... ClaferMooVisualizer> done!
```

Unfortunately, the output view is kind of screwed up (with Undefined's and NaN instead of values)

![mooshot](https://cloud.githubusercontent.com/assets/11213364/6850260/6132a9b2-d3d8-11e4-8169-b02c955b6611.png)

At first, I though this had something to do with the changed output format of claferchocoig between 0.3.6.1 and 0.3.9 [*], which seems to be (intentionally?) caused by https://github.com/gsdlab/ClaferChocoIG/commit/1b95f3d2bad5a302c6b6d34642c4e05d71119ce3

Unfortunately, modifying the code such that the output of both versions is the same again does not seem to fix the issue :-(

Any (better) ideas?

Thx,

Klaas

[*] This is 0.3.9 (as is)
```
c0_aPhone$0 : c0_MobilePhone
        c0_hardware$0 : c0_Feature
                c0_screen$0 : c0_Feature
                        c0_material$0 : c0_Feature
                                c0_amoled$0 : c0_Feature
                                        c0_cost$6 : clafer  =  c0_cost$6
```

This is 0.3.6 (more readable)
```
c0_aPhone$0 : c0_MobilePhone
        c0_hardware$0 : c0_Feature
                c0_screen$0 : c0_Feature
                        c0_material$0 : c0_Feature
                                c0_amoled$0 : c0_Feature
                                        c0_cost$6 : clafer  =  6
```
Note the difference in the last line...

**Comment by mantkiew** [comment]
The visualizer and configurator have not been released for 0.3.9 yet. We're working on adjusting the code to work with the new Clafer (specifically, Clafer now supports having both a super and a reference at the same time, the intermediate representation and XML have changed). The only web tool which works with 0.3.9 currently is the [IDE testing](http://t3-necsis.cs.uwaterloo.ca:8194/). But it has not been released yet.

**Comment by mantkiew** [comment]
These issues are resolved now. We replaced the bad XML output with the always consistent JSON output (it is an automatic serialization of the compiler's intermediate representation).
